homeassistant/components/yale_smart_alarm/__init__.py

```diff
@@ -1 +1,57 @@
 """The yale_smart_alarm component."""
+from __future__ import annotations
+
+import asyncio
+
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN, LOGGER
+from .coordinator import YaleDataUpdateCoordinator
+
+PLATFORMS = ["alarm_control_panel"]
+
+
+async def async_setup_entry(hass, entry):
+    """Set up Yale from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+    title = entry.title
+
+    coordinator = YaleDataUpdateCoordinator(hass, entry=entry)
+
+    await coordinator.async_refresh()
```

Comment:
Use this:
https://github.com/home-assistant/core/blob/7b7062dded1e78db81c7e974330f67a9d176c6e9/homeassistant/helpers/update_coordinator.py#L149